.github/workflows/scripts/scanner-update-nvd.py

```diff
@@ -102,9 +115,11 @@ def dirpath(s):
             for start, end in get_dates(year, 3):# fetch quarterly
                 vulnerabilities.extend(fetch_data(start, end))
 
-            file_path = os.path.join(args.dirpath, f"{year}.json")
+            file_path = os.path.join(args.dirpath, f"{year}.nvd.json")
```

Comment:
Don't have to do this now, but it might be less tech debt if you create a "loader" similar to the one in https://github.com/stackrox/stackrox/blob/master/.github/workflows/scripts/scanner-update-nvd-legacy.py#L66-L83

Reply:
Yeah, I think if you just use the `fetch_data()` like this (not exactly like this, but in the lines of):

```python
class APILoader:

    def fetch(self, year):
        for start, end in get_dates(year, 3):  
                for vulnerability in fetch_data(start, end):
                        yield from fetch_data(start, end)
```